src/languageservice/serviceDownloadProvider.ts

```diff
@@ -99,11 +99,14 @@ export default class ServiceDownloadProvider {
             this._logger.appendLine(`${Constants.serviceInstallingTo} ${installDirectory}.`);
             const urlString = this.getGetDownloadUrl(fileName);
 
+            const isZipFile: boolean = path.extname(fileName) === '.zip';
```

Comment:
nit: but I'd suggest just having this be fileExt and have the calling code determine what they want to do with it. Just a bit more generic (there's more archive types than just zip/tar)